pkgs/applications/office/watson/default.nix

```diff
@@ -14,7 +14,15 @@ buildPythonApplication rec {
 
   checkPhase = ''
     pytest -vs tests
- '';
+  '';
+
+  postInstall = ''
+    mkdir -p $out/share/bash-completion/completions
+    mv watson.completion $out/share/bash-completion/completions/watson
+
+    mkdir -p $out/share/zsh/site-functions/
+    mv watson.zsh-completion $out/share/zsh/site-functions/_watson
```

Comment:
there are some dedicated utilities for this like `installShellFiles` if you are interested.

Reply:
thanks, the PR is using `installShellFiles` now